docker/mint/docker-run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -ex
+export RUST_BACKTRACE=full
```

Comment:
dial this back to 1?

Reply:
Doesn't this just increase the verbosity during panics or is there some additional information being spit out during normal runtime? The documentation is a bit ... sparse on this topic.

Reply:
This is a good discussion to have, but honestly i saw very little difference between 1 and full in terms of lines. Considering how rare this is I'll probably just leave it as full.